TACC/Core-Portal
# Make 3rd party integrations more dynamic

## Overview: ##

- In anticipation of more 3rd party integrations like UT Box, make the integration population in the Manage Account page more dynamic.
- Fix the disabling logic of 3rd party integrations by integrations to correctly account for the integrations the portal has enabled. This is done by adding the `integration` field to a data files dict in `_PORTAL_DATAFILES_STORAGE_SYSTEMS`
- Change the disabling logic of App licenses to be based on the value of `hideApps`

## Related Jira tickets: ##

* [FP-212](https://jira.tacc.utexas.edu/browse/FP-212)
* [FP-1171](https://jira.tacc.utexas.edu/browse/FP-1171)

## Testing Steps: ##
1. In `settings_default.py`, set `hideApps` to `true`
2. Go to https://cep.dev/workbench/account
3. Confirm that the Licenses component is not rendered
4. Confirm that the Google Drive integration works as normal
5. In `settings_default.py`, delete the `Google Drive` dict from `_PORTAL_DATAFILES_STORAGE_SYSTEMS`
6. Confirm the 3rd Party Apps component is not rendered

## UI Photos:
<img width="594" alt="Screen Shot 2021-09-08 at 3 20 10 PM" src="https://user-images.githubusercontent.com/20326896/132584852-a26997be-8a8e-4f6a-9bd0-c4bcf79afb16.png">


## Notes: ##
TODO:
